packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -63,12 +63,44 @@ class Autocomplete<T extends Object> extends StatelessWidget {
     required this.optionsBuilder,
     this.displayStringForOption = RawAutocomplete.defaultStringForOption,
     this.fieldViewBuilder = _defaultFieldViewBuilder,
```

Comment:
Actually now I'm having second thoughts. The point of focusNode and textEditingController in RawAutocomplete is to use a separate field instead of fieldViewBuilder. That seems to not be possible with Autocomplete. What do you want to use these parameters for if not that?